controller.go

```diff
@@ -192,7 +200,7 @@ func main() {
 
 	awsAdapter, err = aws.NewAdapter(controllerID)
 	if err != nil {
-		log.Fatal(err)
+		log.Error(err)
```

Comment:
This should Fatal, because of restarting automatically and trying to get AWS credentials from cache again